core/src/test/java/hudson/util/SecretUtilTest.java

```diff
@@ -0,0 +1,23 @@
+package hudson.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+
+public class SecretUtilTest {
```

Comment:
SecretUtilTest?

Reply:
Yeah, just "SecretTest" would be fine

Reply:
@daniel-beck @oleg-nenashev I had problems with https://github.com/jenkinsci/jenkins/blob/master/core/src/test/groovy/hudson/util/SecretTest.groovy when I named it as SecretTest.java

That's why I use SecretUtilTest.java